app/grandchallenge/cases/image_builders/tiff.py

```diff
@@ -409,26 +414,21 @@ def format_error(message):
             continue
 
         image = _create_tiff_image_entry(tiff_file=gc_file)
-        new_image_files = _add_image_files(
-            gc_file=gc_file, image=image, new_image_files=new_image_files,
-        )
+        new_image_files |= _new_image_files(gc_file=gc_file, image=image,)
+        new_folders |= _new_folder_uploads(dzi_output=dzi_output, image=image,)
+
+        new_images.add(image)
+        consumed_files.add(gc_file.path)
 
-        new_folder_upload = _add_folder_uploads(
-            dzi_output=dzi_output,
-            image=image,
-            new_folder_upload=new_folder_upload,
-        )
-        new_images.append(image)
-        consumed_files.append(gc_file.path)
         if gc_file.associated_files:
             consumed_files += list(f for f in gc_file.associated_files)
```

Comment:
This needs to be converted to append set as well

Reply:
Thanks, done!